app/utils/IncomeSourcesUtils.scala

```diff
@@ -40,9 +42,39 @@ trait IncomeSourcesUtils extends FeatureSwitching {
     }
   }
 
+  def newWithIncomeSourcesRemovedFromSession(redirect: Result, sessionService: SessionService, errorRedirect: Result)(implicit user: MtdItUser[_], ec: ExecutionContext): Future[Result] = {
+    val incomeSourcesSessionKeys = Seq(
+      "addUkPropertyStartDate",
```

Comment:
all these keys defined in SessionKey singleton, pls re-use existing name;
or better would be to defined this sequence in SessionKey and re-use it here

Reply:
The list of session keys in SessionKeys.scala is not complete, it misses several, for example       "addUkPropertyStartDate",
      "addForeignPropertyStartDate",

Reply:
I have moved the list into SessionKeys.scala